### DISCUSSION TITLE
Preserve entity state instead of unknown/unavailable

### DISCUSSION THREAD

**Discussion by CFenner** [home-assistant/frontend#13198] [Dashboard]
I would like to see the *last known value* for some entities. 

For example I have some devices having a battery. Some of them are always on (iPhone), others go into deep sleep to save battery load (IKEA) and others are integrated using a bridge which I sometimes disconnect to save energy (Netatmo). 

For those entities it would be nice to see the last reported state instead of unknown / unavailable. 

![image](https://user-images.githubusercontent.com/9592452/179088138-e6e0097d-3a93-47cf-bfd7-b78412fca8db.jpeg)

**Comment by KTibow** [comment]
You might be able to do this with an `input_number` and an automation.
